iModelCore/ECDb/ECDb/ECSql/ValueExp.cpp

```diff
@@ -915,6 +960,11 @@ Utf8String MemberFunctionCallExp::_ToString() const
 //+---------------+---------------+---------------+---------------+---------------+------
 bool MemberFunctionCallExp::_TryDetermineParameterExpType(ECSqlParseContext& ctx, ParameterExp& parameterExp) const
     {
+    if(this->m_tableValuedFunc && m_functionName.EqualsIAscii("IdSet"))

```

Comment:
Same, use IsTableValuedFunc() here, also, we should put "IdSet" into a const/define somewhere instead of hard typing it in multiple spots

Reply:
done

Reply:
removed hard coded string as well